test/e2e/network_policy.go

```diff
@@ -418,13 +434,10 @@ func createNetworkClientPod(f *framework.Framework, namespace *v1.Namespace, pod
 			Containers: []v1.Container{
 				{
 					Name:  fmt.Sprintf("%s-container", podName),
-					Image: "gcr.io/google_containers/busybox:1.24",
+					Image: "gcr.io/google_containers/alpine-with-bash:1.0",
 					Args: []string{
-						"/bin/wget",
-						"-T", "8",
-						fmt.Sprintf("%s.%s:%d", targetService.Name, targetService.Namespace, targetPort),
-						"-O",
-						"-",
+						fmt.Sprintf("for i in {1..5}; do wget -T 8 %s.%s:%d -O - && exit 0 || sleep 1; done; exit 1",
```

Comment:
You've added a readiness probe to the server pod now, so we should just not run the client pod until the server pod is ready.